lang/ru/lang.php

```diff
@@ -5,22 +5,40 @@
  * 
  * @author Aleksandr Selivanov <alexgearbox@gmail.com>
  * @author Vitaly Filatenko <kot@hacktest.net>
+ * @author Rainbow-Spike <rainbow_spike@derpy.ru>
  */
+
+// banner
```

Comment:
Usually these comments and blank lines should be removed.

Reply:
nope, better to not delete this